static2/src/services/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: 'http://127.0.0.1:5000/', 
```

Comment:
Looks like a another duplicate 🤔 remember to use `window.location.origin`, if it's not